src/workerd/jsg/jsg.h

```diff
@@ -945,15 +945,17 @@ struct Dict {
     Value value;
 
     JSG_MEMORY_INFO(Field) {
-      tracker.trackField(name);
-      tracker.trackField(value);
+      tracker.trackField("name", name);
+      tracker.trackField("value", value);
```

Comment:
Maybe "key" and "value" would be slightly more standard?

Reply:
possibly but I want to keep the names aligned with the actual fields as much as possible.